pkg/api/graph/graph.go

```diff
@@ -508,7 +508,7 @@ func ReverseGraphEdge(g Interface, from, to graph.Node, edgeKinds sets.String) b
 // AddReversedEdge adds a reversed edge for every passed edge and preserves the existing
 // edge. Used to convert a one directional edge into a bidirectional edge, but will
 // create duplicate edges if a bidirectional edge between two nodes already exists.
-func AddReversedEdge(g Interface, from, to graph.Node, edgeKinds sets.String) bool {
+func AddReversedEdge(g Interface, from, to graph.Node) bool {
```

Comment:
Also, a `grep` shows me I am the *only* consumer of this function. Why should we make sure it's an `EdgeFunc`?